app/src/main/java/com/sunasterisk/musixmatch/ui/playing/lyrics/LyricsContract.java

```diff
@@ -0,0 +1,13 @@
+package com.sunasterisk.musixmatch.ui.playing.lyrics;
+
+public interface LyricsContract {
+    interface Presenter {
+        void getLyrics(String trackName, String artistName);
```

Comment:
sao getLyric lại cần artistName nhỉ

Reply:
cái này là API nó thế ạ. 
Gỉa dụ trong trường hợp có rất nhiều bài hát cùng tên ý ạ, nhưng thực chất nó lại là những bài có nội dung hoàn toàn khác nhau do các ca sĩ khác nhau trình bày ạ. Kể cả cùng 1 bài hát nhưng 2 ca sĩ khác nhau họ thể hiện cũng khác nhau ý ạ.  Nên để khớp với bài hiện tại thì mình sẽ truyền tên của bài hiện tại và ca sĩ hiện tại.